solution/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "solution",
+  "version": "1.0.0",
+  "description": "Robert's solutions",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha",
+    "test-with-coverage": "nyc --reporter=html --reporter=text mocha"
+  },
+  "author": "Robert",
+  "license": "ISC",
+  "devDependencies": {
+    "mocha": "^5.2.0",
+    "nyc": "^13.0.1"
+  }
+}
```

Comment:
How did you generate this file?

Reply:
I run `npm init`, it asked me for most details.
Then I installed and saved _mocha_ test runner, `npm install --save-dev mocha`,
and _instanbul_ for test coverage reports 'npm install --save-dev nyc', which were added to "devDependencies".
Then I added a new script "test-with-coverage" which runs nyc and generates coverage report.
Is there a problem with the file?